pkg/executor/executor.go

```diff
@@ -52,25 +58,17 @@ func (e *MySQLExecutor) EnterTx(options *sql.TxOptions, task func(tx *sql.Tx) er
 	return
 }
 
-func (e *MySQLExecutor) Query(query string) (Rows, error) {
-	var row Rows
-	err := e.EnterTx(nil, func(tx *sql.Tx) error {
-		data, err := tx.Query(query)
-		if err != nil {
-			return err
-		}
-
-		row, err = NewRows(data)
-		if err != nil {
-			return err
-		}
-		return nil
-	})
-	return row, err
+func (e *MySQLExecutor) Query(query string) (rows Rows, err error) {
+	data, err := e.db.Query(query)
+	if err != nil {
+		return
+	}
+	rows, err = NewRows(data)
+	return
 }
 
-func (e *MySQLExecutor) Exec(query string) (result Result, err error) {
-	err = e.EnterTx(&sql.TxOptions{Isolation: sql.LevelDefault}, func(tx *sql.Tx) error {
+func (e *MySQLExecutor) ExecAndRollback(query string) (result Result, err error) {
```

Comment:
This is odd for me, do we really need it?

Reply:
We need it to support insert, update and delete statements.